src/crypto/hsm/nxp/CHIPCryptoPALHsm_SE05X_P256.cpp

```diff
@@ -182,10 +183,15 @@ CHIP_ERROR P256KeypairHSM::ECDSA_sign_msg(const uint8_t * msg, size_t msg_length
     status = sss_asymmetric_context_init(&asymm_ctx, &gex_sss_chip_ctx.session, &keyObject, kAlgorithm_SSS_SHA256, kMode_SSS_Sign);
     VerifyOrExit(status == kStatus_SSS_Success, error = CHIP_ERROR_INTERNAL);
 
-    status = sss_asymmetric_sign_digest(&asymm_ctx, hash, hashLen, Uint8::to_uchar(out_signature), &siglen);
+    status = sss_asymmetric_sign_digest(&asymm_ctx, hash, hashLen, signature_se05x, &signature_se05x_len);
     VerifyOrExit(status == kStatus_SSS_Success, error = CHIP_ERROR_INTERNAL);
 
-    SuccessOrExit(out_signature.SetLength(siglen));
+    VerifyOrExit(CHIP_NO_ERROR ==
+                     EcdsaAsn1SignatureToRaw(kP256_FE_Length, signature_se05x, signature_se05x_len, out_signature.Bytes(),
```

Comment:
Please use the CHIPCryptoPAL.h primitives:

- `EcdsaRawSignatureToAsn1`
- `EcdsaAsn1SignatureToRaw`

Reply:
The format of the arguments changed:
```
CHIP_ERROR EcdsaAsn1SignatureToRaw(size_t fe_length_bytes, const ByteSpan & asn1_sig, MutableByteSpan & out_raw_sig);
```

Reply:
Updated the PR with the required changes